node-notes-cli/app.js

```diff
@@ -0,0 +1,95 @@
+const fs = require('fs');
+
+const keyWord = process.argv[2];
+const keyWordLib = ['read', 'create', 'update', 'delete'];
+
+if (!keyWordLib.includes(keyWord)) {
+  console.error({ error: 'invalid command' });
+} else if (keyWordLib.includes(keyWord)) {
+  if (keyWord === 'read') {
+    fs.readFile('data.json', 'utf8', (err, data) => {
+      if (err) {
+        console.error(err);
+      }
```

Comment:
If there's an error reading the file, it'd make sense to stop attempting to process its contents.